### PR TITLE
Update Makefile

### PR DESCRIPTION
**Pull request by yaoyaoyao480**
udshiufshdfiuh siufhiusfhiushiu

### CODE REVIEW

File: QR-Code-generator-master/cpp/Makefile [hunk 1]
```diff
@@ -13,14 +13,14 @@
 # - The above copyright notice and this permission notice shall be included in
 #   all copies or substantial portions of the Software.
 # - The Software is provided "as is", without warranty of any kind, express or
```
Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:15]: 33333333333333333

File: QR-Code-generator-master/cpp/Makefile [hunk 2]
```diff
@@ -13,14 +13,14 @@
 # - The above copyright notice and this permission notice shall be included in
 #   all copies or substantial portions of the Software.
```
Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:14]: 不是吧不是吧不是吧不是吧

**Review by yaoyaoyao480 — COMMENTED** [review]
整体不错棒棒哒

**Review by yaoyaoyao480 — COMMENTED** [review]
good job

File: QR-Code-generator-master/cpp/Makefile [hunk 3]
```diff
@@ -13,14 +13,14 @@
 # - The above copyright notice and this permission notice shall be included in
```
Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:13]: <i class='graphite__hidden'>[Re: lines 10 to 10]<br/><br/></i>

评论评论评论<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/StephanieLeeeee/Winterfall/5?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by StephanieLeeeee — APPROVED** [review]

**Review by StephanieLeeeee — APPROVED** [review]

**Review by StephanieLeeeee — CHANGES_REQUESTED** [review]
111

**Review by yaoyaoyao480 — COMMENTED** [review]
ghhuhuhuuhuhu

**Review by yaoyaoyao480 — COMMENTED** [review]
测试评论

**Comment by yaoyaoyao480** [comment]
### Merge activity

* **Jan 29, 11:32 AM**: @yaoyaoyao480 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/StephanieLeeeee/Winterfall/5).

**Review by yaoyaoyao480 — COMMENTED** [review]
评论

File: QR-Code-generator-master/cpp/Makefile (hunk 3 above)
Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:13]: <i class='graphite__hidden'>[Re: lines 7 to 7]<br/><br/></i>

Added via Giphy<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/StephanieLeeeee/Winterfall/5?utm_source=unchanged-line-comment">Graphite</a>.</span>

File: QR-Code-generator-master/cpp/Makefile
Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:1]: 评论

File: QR-Code-generator-master/cpp/Makefile (hunk 3 above)
Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:13]: <i class='graphite__hidden'>[Re: lines 7 to 7]<br/><br/></i>

评论<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/StephanieLeeeee/Winterfall/5?utm_source=unchanged-line-comment">Graphite</a>.</span>

Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:13]: <i class='graphite__hidden'>[Re: lines 3 to 3]<br/><br/></i>

评论<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/StephanieLeeeee/Winterfall/5?utm_source=unchanged-line-comment">Graphite</a>.</span>

Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:13]: <i class='graphite__hidden'>[Re: lines 5 to 5]<br/><br/></i>

发表一条意见吧<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/StephanieLeeeee/Winterfall/5?utm_source=unchanged-line-comment">Graphite</a>.</span>

Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:13]: <i class='graphite__hidden'>[Re: lines 1 to 1]<br/><br/></i>

发表第二条意见<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/StephanieLeeeee/Winterfall/5?utm_source=unchanged-line-comment">Graphite</a>.</span>

Comment (yaoyaoyao480) [inline QR-Code-generator-master/cpp/Makefile:13]: <i class='graphite__hidden'>[Re: lines 2 to 2]<br/><br/></i>

发表第三条意见<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/StephanieLeeeee/Winterfall/5?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by yaoyaoyao480 — COMMENTED** [review]
总体来说是不错的hh

**Pull request merged**